dist/panel-dialog/panel-dialog.css

```diff
@@ -35,6 +37,10 @@
   flex-shrink: 0;
   margin: 16px 16px 0;
   position: relative;
+  height: 32px;
+  /* stylelint-disable-next-line declaration-block-no-duplicate-properties */
+  position: fixed;
+  width: calc(100% - 32px);
```

Comment:
I don't think these are needed. I removed them locally and it was working just fine. 
Just curious why these were added?

Reply:
Weird. When I added it, it wouldn't work without them, but now it does appear to work without it. Maybe I added something in between? Dunno. I'll remove them.